nym-vpn-apple/Settings/Sources/Settings/SettingsFlowCoordinator.swift

```diff
@@ -3,6 +3,8 @@ import AppSettings
 import NymLogger
 
 struct SettingsFlowCoordinator<Content: View>: View {
+    @EnvironmentObject private var logFileManager: LogFileManager
```

Comment:
Nit: You might want to make it abstract protocol to avoid making too much of LogFileManager public to coordinators. But this is minor.